app/code/Magento/Sales/Model/Order.php

```diff
@@ -393,6 +399,7 @@ public function __construct(
         \Magento\Sales\Model\ResourceModel\Order\CollectionFactory $salesOrderCollectionFactory,
         PriceCurrencyInterface $priceCurrency,
         \Magento\Catalog\Model\ResourceModel\Product\CollectionFactory $productListFactory,
+        OrderStateResolverInterface $orderStateResolver,
```

Comment:
Please follow Backward Compatibility for adding new constructor params.
https://devdocs.magento.com/contributor-guide/backward-compatible-development/#php

Reply:
done. Thank you